pkg/controller/controller_binding.go

```diff
@@ -400,7 +400,24 @@ func (c *controller) injectBinding(binding *v1alpha1.Binding, credentials map[st
 		},
 	}
 
-	_, err = c.kubeClient.SettingsV1alpha1().PodPresets(binding.Namespace).Create(podPreset)
+	found = false
+	_, err = c.kubeClient.SettingsV1alpha1().PodPresets(binding.Namespace).Get(podPreset.ObjectMeta.Name, metav1.GetOptions{})
+	if err == nil {
+		found = true
+	}
```

Comment:
@ash2k agreed, I will probably refactor the whole method as a separate pull request